Pickers/ActionSheetDatePicker.h

```diff
@@ -48,31 +48,45 @@ typedef void(^ActionDateCancelBlock)(ActionSheetDatePicker *picker);
 @property (nonatomic, copy) ActionDateDoneBlock onActionSheetDone;
 @property (nonatomic, copy) ActionDateCancelBlock onActionSheetCancel;
 
-+ (instancetype)showPickerWithTitle:(NSString *)title datePickerMode:(UIDatePickerMode)datePickerMode selectedDate:(NSDate *)selectedDate target:(id)target action:(SEL)action origin:(id)origin;
-
-+ (instancetype)showPickerWithTitle:(NSString *)title datePickerMode:(UIDatePickerMode)datePickerMode selectedDate:(NSDate *)selectedDate target:(id)target action:(SEL)action origin:(id)origin cancelAction:(SEL)cancelAction;
++ (instancetype)showPickerWithTitle:(NSString *)title
```

Comment:
Would be nice to pass the datePickerStyle. Default could be .automatic.

Reply:
Would you prefer converting this class to Swift or do the `datePickerStyle` changes in the current Objective C class?

Reply:
I think Objective C should be enough in this case.